x-pack/plugins/security_solution/public/common/components/exceptions/use_fetch_or_create_rule_exception_list.test.tsx

```diff
@@ -28,7 +28,8 @@ const mockKibanaHttpService = coreMock.createStart().http;
 jest.mock('../../../detections/containers/detection_engine/rules/api');
 jest.mock('@kbn/securitysolution-list-api');
 
-describe('useFetchOrCreateRuleExceptionList', () => {
+// Broken during Jest 29 upgrade (also skipped on main see https://github.com/elastic/kibana/issues/145749)
+describe.skip('useFetchOrCreateRuleExceptionList', () => {
```

Comment:
This has an outstanding issue #145749 when `main` upgraded Jest as well.

Reply:
For this one I think we should update the issue and also include the label 7.17.11 to indicate this was also skipped in another branch